Tests/UnitTests/Mocks/MockSystemInfo.swift

```diff
@@ -78,7 +78,7 @@ class MockSystemInfo: SystemInfo {
 
 extension MockSystemInfo: @unchecked Sendable {}
 
-extension RevenueCat.OperatingSystemVersion: Swift.Comparable {
```

Comment:
I remember needing to do this change but I'm not sure if it'd break other things... However, there isn't any `OperatingSystemVersion` defined in RevenueCat, so this should have never been allowed in the first place. I don't understand 😅

Reply:
I think this file should not be included in the project, but let's tackle that later